plugin/azure/azure.go

```diff
@@ -13,53 +14,72 @@ import (
 	"github.com/coredns/coredns/plugin/pkg/upstream"
 	"github.com/coredns/coredns/request"
 
-	azuredns "github.com/Azure/azure-sdk-for-go/profiles/latest/dns/mgmt/dns"
+	publicAzureDNS "github.com/Azure/azure-sdk-for-go/profiles/latest/dns/mgmt/dns"
```

Comment:
you can use caps in imports , they look also a bit ugly, what the default import path here, just 'dns'?

Reply:
we have 2 `dns` now, `publicdns` and `privatedns`. They are different packages and both need importing hence `publicAzureDNS`. 
Should I change it to `publicdns` and `privatedns`?